tv-show/src/components/features/review/ReviewItem/ReviewItem.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { IReview } from "@/typings/Review.type";
+import { render, screen } from "@testing-library/react";
+import ReviewItem from "./ReviewItem";
+
+describe('ReviewItem', () => {
+  const email = 'test@jest.js'
+  const rating = 3;
+  const comment = 'test comment';
+  const review: IReview = { email, rating, comment };
+
+  it('should render the correct user email', () => {
+    render(<ReviewItem review={review} onRemoveClick={(review) => null}/>);
+
+    expect(screen.getByText(email)).toBeInTheDocument();
+    try {
```

Comment:
Why the try catch? Don't you want the tests to fail when needed?

Reply:
This try-catch is for making sure that some "random" text is not rendered on the screen. I'm doing a `getByText()` with a non-existing text, and I expect that to throw an error. If it throws, that means that there is no random text.

What I wanted to test here is that the getByText won't catch any substrings (if I say `getByText(email+'x')` I don't want it to find something just because `email` is gettable by text).

Maybe it's testing the testing framework - that's an overkill, and I can, ofc, remove it from the test